docs/LocalityAwareness.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Locality Awareness In LoadAwareShuffleGrouping
+layout: documentation
+documentation: true
+---
+
+# Locality Awareness In LoadAwareShuffleGrouping
+
+### Motivation
+
+Apache Storm 2.0 introduced locality awareness to LoadAwareShuffleGrouping based on Bang-Bang control theory. 
```

Comment:
I would prefer to change to "Storm has introduced" or something instead?  As time goes by, this version becomes irrelevant.  I think it's understood that not all features are available in older Storm versions.  I'm ok with it if you think it's necessary though.